CHANGELOG.md

```diff
@@ -90,6 +90,12 @@
 
 * Fix MDN links to static interface methods.
   [#4010](https://github.com/rustwasm/wasm-bindgen/pull/4010)
+ 
+* Fixed Deno support.
+  [#3990](https://github.com/rustwasm/wasm-bindgen/pull/3990)
+
+* Fixed Deno support.
+  [#3990](https://github.com/rustwasm/wasm-bindgen/pull/3990)
```

Comment:
```suggestion
```
Changelog entry was accidentally duplicated.

Reply:
I have to rebase twice, because conflicts on it kept showing up, although I was careful, not sure how that passed. Thanks!